rdf4j/src/main/java/org/apache/commons/rdf/rdf4j/RDF4JQuad.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.rdf.rdf4j;
+
+import org.apache.commons.rdf.api.Quad;
+
+/**
+ * Marker interface for RDF4J implementations of Quad.
+ * 
+ */
+public interface RDF4JQuad extends Quad, RDF4JTripleLike {
```

Comment:
Good that RDF4J goes "quad first"!

That would simplify a lot both here and in jsonld-java integration, I agree. However `Triple` is where we started and also a prime type in Jena. RDF 1.1 does not really define a 'quad' as much as it defines a "triple". Also many non-expert users are likely to just want to stay at the RDF graph level and could be confused if their 'contains' call fails.

And so I think we should still support triples and graphs for the RDF4J integration.